cypress/e2e/components/generator.spec.js

```diff
@@ -189,30 +195,64 @@ describe('Request generator tool component', () => {
         }
     });
 
-    it('loads company from slug and clears URL parameters afterwards', () => {
-        cy.visit('/generator#!company=airbnb');
-        cy.contains('Get access').click();
-        cy.contains('Fill in request to “Airbnb Ireland UC”');
+    // `cy.generatorStore()` still doesn't work in CI, unfortunately.
```

Comment:
As @mal-tee said, you should link to https://github.com/datenanfragen/website/issues/1043#issuecomment-1472355575 here.